src/components/contests/contestShow.jsx

```diff
@@ -15,14 +15,18 @@ class ContestShow extends React.Component {
   }
 
   render() {
-    const { contest, entries } = this.props;
+    const { contest, entries, countries } = this.props;
```

Comment:
so this is taking the props object and extracting contest, entries, and countries into constants of the same name, yeah? been a bit since we reviewed this in class, so just confirming...

Reply:
Yes, exactly!